chainerx_cc/chainerx/routines/loss.cc

```diff
@@ -0,0 +1,28 @@
+#include "chainerx/routines/loss.h"
+
+#include "chainerx/array.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/routines/explog.h"
+#include "chainerx/routines/indexing.h"
+#include "chainerx/routines/misc.h"
+#include "chainerx/scalar.h"
+
+namespace chainerx {
+
+Array MeanAbsoluteError(const Array& x1, const Array& x2) { return Absolute(x1 - x2).Mean(); }
+
+Array MeanSquaredError(const Array& x1, const Array& x2) { return SquaredDifference(x1, x2).Mean(); }
+
+Array GaussianKLDivergence(const Array& mu, const Array& ln_var) { return (Square(mu) + Exp(ln_var) - ln_var - 1) * 0.5; }
```

Comment:
A detail but let's be consistent with `mu` and `mean`. I think the latter is better as it's used in Chainer but open for discussions.

Reply:
I agree.